src/amazon-marketplace.ts

```diff
@@ -90,6 +91,7 @@ export enum AmazonMarketplaceAdvertisingTimeZone {
   EUROPE_LONDON = 'Europe/London',
   EUROPE_PARIS = 'Europe/Paris',
   EUROPE_STOCKHOLM = 'Europe/Stockholm',
+  EUROPE_WARSAW = 'Europe/Warsaw',
```

Comment:
Is this documented somewhere? I couldn't find it.

Reply:
@moltar : You can find it out here: 👍 
![image](https://user-images.githubusercontent.com/19872073/148873146-49d95749-367c-4a6a-b7c9-0f2aa1664c73.png)

Docs (Gets a list of profiles > Response > Timezone): https://advertising.amazon.com/API/docs/en-us/reference/2/profiles#:~:text=Europe/Stockholm-,EU,Europe/Warsaw,-EU

Reply:
Nice one, @nguyentoanit!